### PR TITLE
For Footer and buttons in mobile view is been set , so that they dont conjest .

### PR DESCRIPTION
**Pull request by syedjafer**
For Footer and buttons in mobile view is been set , so that they dont conjest .

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
   <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700,800,400italic,300,300italic,700italic,800italic' rel='stylesheet' type='text/css'>
 
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (astronomersiva) [inline _includes/head.html]: Can you fetch only the required fonts? Maybe even svgs from somewhere is fine. Let's not fetch the entire font-awesome library here

File: css/pysangamam.css [hunk 2]
```diff
@@ -123,3 +123,18 @@ h1#title:hover span.mobile-span:after {
 #map {
   height: 500px;
 }
+
+#footer{
+  padding-top: 0 !important;
```
Comment (astronomersiva) [inline css/pysangamam.css]: Can you try and see if `!important` can be removed from all these styles?

File: css/pysangamam.css [hunk 3]
```diff
@@ -123,3 +123,18 @@ h1#title:hover span.mobile-span:after {
...
+  margin-top: 40px !important;
+  background: #f3f3f3;
+  position: absolute; 
+  width:100%;
+  color:black !important;
+  border-bottom: 20px solid #218c74;
```
Comment (astronomersiva) [inline css/pysangamam.css]: 15px is what we use everywhere for this bottom border.

**Review by astronomersiva — CHANGES_REQUESTED** [review]

File: _includes/footer.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+				<div class="col-xs-12 col-sm-12 col-md-12 mt-2 mt-sm-5">
+					<ul class="list-unstyled list-inline social text-center">
+						<li class="list-inline-item"><a href="https://twitter.com/PySangamam"><img src="img/twitter.png" alt="twitter"></a></li>
+						<li class="list-inline-item"><a href="mailto:chennaipy@python.org" target="_blank"><img src="img/envelope.png" alt="Mail Me"></a></li>
+					</ul>
+				</div>
+			</div>	
+			<div class="row">
+				<div class="col-xs-12 col-sm-12 col-md-12 mt-2 mt-sm-2 text-center ">
+					<p>Brought to you by<a class="ml-2" href="https://chennaipy.org">Chennaipy .</a> Text is available under the<a class="ml-2" href="https://creativecommons.org/licenses/by-sa/3.0/" target="_blank" data-toggle="tooltip" data-placement="top" title="Creative Commons Attribution-ShareAlike License."><i class="fa fa-creative-commons"></i> Creative Commons Attribution-ShareAlike License.</a></p>
```
Comment (astronomersiva) [inline _includes/footer.html]: Chennaipy . - There shouldn't be a space between Chennaipy and the full stop.

File: index.html [hunk 5]
```diff
@@ -175,3 +175,5 @@ <h2 class="text-center"><b>Location</b></h2>
     </div>
   </div>
 </div>
+
+{% include footer.html %}
```
Comment (astronomersiva) [inline index.html]: Can you add this to the default and page templates instead of here? That way, all pages that use that template will have the footer.

Comment (syedjafer) [reply index.html]: :thumbsup:

**Review by astronomersiva — APPROVED** [review]

**Comment by astronomersiva** [comment]
@syedjafer Thanks a lot for this 🎉

**Pull request merged**